larray/excel.py

```diff
@@ -83,8 +111,9 @@ def __init__(self, filepath=None, overwrite_file=False, visible=None, silent=Non
                     app = xw_wkb.app
 
             # active workbook use active app by default
-            if filepath == -1 and app is None:
-                app = -1
+            # XXX: I wonder if we have a choice here (ie maybe the check for app == "global" is superfluous)
+            if filepath == -1 and app == "global":
+                app = "active"
```

Comment:
Yes, I think it is superfluous. We can return an error with an explicit message if filepath == -1 and app != "global"?